opentelemetry-sdk/tests/metrics/test_measurement_consumer.py

```diff
@@ -13,87 +13,78 @@
 # limitations under the License.
 
 from unittest import TestCase
-from unittest.mock import Mock, patch
+from unittest.mock import MagicMock, Mock, patch
 
-from opentelemetry.sdk._metrics import MeterProvider
 from opentelemetry.sdk._metrics.measurement_consumer import (
     MeasurementConsumer,
     SynchronousMeasurementConsumer,
 )
+from opentelemetry.sdk._metrics.point import AggregationTemporality
+from opentelemetry.sdk._metrics.sdk_configuration import SdkConfiguration
 
 
+@patch("opentelemetry.sdk._metrics.measurement_consumer.MetricReaderStorage")
 class TestSynchronousMeasurementConsumer(TestCase):
-    def test_parent(self):
+    def test_parent(self, _):
 
         self.assertIsInstance(
-            SynchronousMeasurementConsumer(), MeasurementConsumer
+            SynchronousMeasurementConsumer(MagicMock()), MeasurementConsumer
         )
 
-    @patch("opentelemetry.sdk._metrics.SynchronousMeasurementConsumer")
```

Comment:
I just moved these to `test_metrics.py` because they are testing MeterProvider's use of MeasurementConsumer, not the consumer itself.